src/lib/y2users/users_simple/reader.rb

```diff
@@ -20,13 +20,21 @@
 require "yast"
 require "y2users/user"
 require "y2users/password"
+require "y2users/parsers/shadow"
 
 Yast.import "UsersSimple"
 
 module Y2Users
   module UsersSimple
     # Class for reading users configuration from old Yast::UsersSimple module
     class Reader
+      # @see #{shadow_string}
+      SORTED_SHADOW_ATTRS = [
+        "uid", "userPassword", "shadowLastChange", "shadowMin", "shadowMax",
```

Comment:
I would write here that uid is actually username as it is confusing as shadow does not contain uid.